### PR TITLE
[Task] VX-272: Use table to list the nodes.

### PR DESCRIPTION
**Pull request by WakeupTsai**

### CODE REVIEW

File: src/routes/Cluster/Node/index.tsx [hunk 1]
```diff
@@ -292,30 +333,96 @@ class Node extends React.Component<NodeProps, NodeState> {
...
-      <Card
-        title={this.props.nodes[node].detail.hostname}
-        extra={<a onClick={() => this.showMore(node)}>More</a>}
-      >
-        {this.renderDetail(node)}
-      </Card>
+      numerator.toString() +
+      ' (' +
+      ((numerator / denominator) * 100).toFixed(2).toString() +
+      '%)'
```
Comment (chenyunchen) [inline src/routes/Cluster/Node/index.tsx]:
```
`${numerator.toString()} (${((numerator / denominator) * 100).toFixed(2).toString()}%)`
```

File: src/routes/Cluster/Node/index.tsx [hunk 2]
```diff
@@ -292,30 +333,96 @@ class Node extends React.Component<NodeProps, NodeState> {
...
+        this.props.nodes[node].resource.memoryRequests,
+        this.props.nodes[node].resource.capacityMemory
+      ),
+      memoryLimits: this.calculatePercentByte(
+        this.props.nodes[node].resource.memoryLimits,
+        this.props.nodes[node].resource.capacityMemory
+      ),
+      age: moment(
+        new Date(this.props.nodes[node].detail.createAt * 1000)
+      ).fromNow()
```
Comment (chenyunchen) [inline src/routes/Cluster/Node/index.tsx]: moment(this.props.nodes[node].detail.createAt * 1000).fromNow()

File: src/routes/Cluster/Node/index.tsx [hunk 3]
```diff
@@ -292,30 +333,96 @@ class Node extends React.Component<NodeProps, NodeState> {
...
   };
 
+  protected calculatePercentByte = (numerator: number, denominator: number) => {
+    return (
+      formatBytes(numerator) +
+      ' (' +
+      ((numerator / denominator) * 100).toFixed(2).toString() +
+      '%)'
+    );
+  };
```
Comment (chenyunchen) [inline src/routes/Cluster/Node/index.tsx]:
```
`${formatBytes(numerator)} (${((numerator / denominator) * 100).toFixed(2).toString()}%)`
```

**Review by LucienLee — COMMENTED** [review]
please put 
`const TabPane = Tabs.TabPane;` under interface declaration

File: src/routes/Cluster/Node/index.tsx [hunk 4]
```diff
@@ -85,30 +96,48 @@ class Node extends React.Component<NodeProps, NodeState> {
...
-          this.props.nodes[node].detail.kubeproxyVersion
-        )}
-        {this.renderListItemContent(
-          <FormattedMessage id={`node.detail.kubernetesVersion`} />,
-          this.props.nodes[node].detail.kubernetesVersion
-        )}
+        <Row>
+          <Col span={12}>
+            {this.renderListItemContent(
+              <FormattedMessage id={`node.detail.createdAt`} />,
```
Comment (LucienLee) [inline src/routes/Cluster/Node/index.tsx]: if you don't have concat variable, just use plan single quote rather than literal template.

Comment (LucienLee) [reply src/routes/Cluster/Node/index.tsx]: and also could remove bracket.

**Pull request merged**